### PR TITLE
bradl3yC - 5553 - Address validation warning spacing

### PR DESCRIPTION
**Pull request by bradl3yC**
## Description
-[x] - The warning icon was not respecting margin-left

### Before:
Screen Shot 2020-02-14 at 9 06 04 AM

### After:
Screen Shot 2020-02-14 at 9 05 47 AM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/user-nav/sass/user-nav.scss [hunk 1]
```diff
@@ -269,4 +269,11 @@ span.sidelines {
 
 #help-menu {
   width: 21rem;
+}
+
+#address-validation-warning {
```
Comment (erikphansen) [inline src/platform/site-wide/user-nav/sass/user-nav.scss]: I'm thinking that we should make this change to the AlertBox component itself unless we are sure that this is the only instance of an AlertBox in a modal that has this issue.

Comment (bradl3yC) [reply src/platform/site-wide/user-nav/sass/user-nav.scss]: @erikphansen do we want to make this change for all alert boxes? Or do we want this isolated to the only alert box that (we know of) thats experiencing this issue?

Comment (erikphansen) [reply src/platform/site-wide/user-nav/sass/user-nav.scss]: Yeah the same thing happens in the Direct Deposit payment info modal

Screen Shot 2020-02-14 at 9 47 26 AM

Comment (bradl3yC) [reply src/platform/site-wide/user-nav/sass/user-nav.scss]: @erikphansen Righto, Ill move this over to formation

**Review by erikphansen — APPROVED** [review]

**Comment by bradl3yC** [comment]
@department-of-veterans-affairs/frontend-review-group ready for review 👍

**Review by U-DON — APPROVED** [review]

**Pull request merged**